engine/dlib/src/dlib/log.cpp

```diff
@@ -489,6 +525,21 @@ void dmLogInternal(dmLogSeverity severity, const char* domain, const char* forma
 
     va_end(lst);
 
+    if (!g_isSendingLogs)
```

Comment:
This flag is important to prevent infinite loop when call logs from a log listener